tools/build-installer/inno/installer.twig

```diff
@@ -286,13 +285,39 @@ begin
     end;
 end;
 
+procedure InitializeDependencies();
+begin
+  // Add dependencies here
+
+  // .NET Framework 4.8
+  Dependency_AddDotNet48;

```

Comment:
will this correctly handle Windows 10 *not* having to install 4.8 because 4.8.1 is installed? and will this install 4.8.1 if for some reason Windows 10 installation doesn't have it?

Reply:
my point is that the people likely thought of that too